apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -151,6 +151,8 @@ namespace MWMechanics
             static const std::string sIdleSelectToGroupName[GroupIndex_MaxIdle - GroupIndex_MinIdle + 1];
 
             static int OffsetToPreventOvercrowding();
+
+            float fIdleChanceMultiplier;
```

Comment:
Same issue as in #698 - avoid adding fields to AiPackage that don't belong in the package description.